src/sass/components/_header.scss

```
@@ -0,0 +1,206 @@
+
+  &__h1 {
+    display: none;
+    align-items: center;
+    font-family: $FF;
+    font-size: var(--fs-800);
+    font-weight: var(--fs-500);
+    line-height: var(--lh-600);
+    height: 35px;
+    margin-left: 12px;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    color: $font-color-light;
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__main-content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 54px;
+  }
+
+  &__title {
+    display: flex;
+    align-items: center;
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__navigation {
+    display: flex;
+    gap: 40px;
+  }
+
+  &__navigation a {
+    position: relative;
+  }
+
+  &__navigation .active::after {
+    content: '';
+    position: absolute;
+    height: 3px;
+    width: 100%;
+    background-color: red;
+    left: 0;
+    bottom: -3px;
+  }
+
+  &__home {
+    color: $font-color-light;
+    font-family: $FF;
+    font-size: var(--fs-400);
+    font-weight: var(--fs-500);
+    line-height: var(--lh-400);
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__my-library {
+    color: $font-color-light;
+    font-family: $FF;
+    font-size: var(--fs-400);
+    font-weight: var(--fs-500);
+    line-height: var(--lh-400);
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__home:hover,
+  &__my-library:hover,
+  &__home:focus,
+  &__my-library:focus {
+    color: $bg-color-accent;
+  }
+  &__search-input:hover,
+  &__search-input:focus {
+    border-color: $bg-color-accent;
+  }
+  &__link:hover,
+  &__link:focus {
+    color: $bg-color-accent;
+  }
+
+  &__link:hover svg {
+    fill: $bg-color-accent;
+  }
+
+  &__div-input {
+    display: flex;
+    justify-content: center;
+    transition: color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__icon {
+    fill: $bg-color-accent;
+    width: 24px;
+    height: 24px;
+  }
+
+  &__icon-search {
+    position: relative;
+    top: 4px;
+    right: 12px;
+    width: 12px;
+    height: 12px;
+  }
+
+  &__search-input {
+    display: flex;
+    align-self: center;
+    margin-left: 12px;
+    width: 280px;
+    background: transparent;
+    background: url('/src/images/icons.svg#icon-film_header');
+    border: none;
+    border-bottom: 0.5px solid #ffffff;
+    outline: none;
+    color: $font-color-light;
+    font-family: $FF;
+    font-size: var(--fs-500);
+    font-weight: var(--fs-400);
+    line-height: var(--lh-300);
+    transition: border-color 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  }
+
+  &__search-input::placeholder {
+    color: $font-color-light;
+    font-family: $FF;
+    font-size: var(--fs-500);
+    font-weight: var(--fs-400);
+    line-height: var(--lh-300);
+    padding-top: 10px;
+  }
+}
+
+@include mq(tablet, and, desktop) {
+  .header {
+    padding: 40px 32px 81px;
+
+    &__h1 {
+      display: flex;
+    }
+
+    &__icon {
+      margin-bottom: 2px;
+    }
```

Ten margin-bottom:2px 

1. Lepiej zamiast margin-bottom napisac transform: translateY: 2px; <- optymalizacja to jest lżejsze dla przeglądarki.
2. I to powinno to być raczej poza `@include mq(tablet, and, desktop) {`
np: 
**Na lini 155 lub 177 dodać to** od 164-166 usunąć
```suggestion
@include mq(tablet) {
  &__icon {
       transform: translateY(2px);
  }
}
```